client/src/pages/Logout.jsx

```diff
@@ -1,14 +1,43 @@
-import React, { useEffect, useContext } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Redirect } from 'react-router-dom';
-import UserContext from '../components/UserContext';
+import Backdrop from '@material-ui/core/Backdrop';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Cookies from 'js-cookie';
+
+import { formStyles } from '../assets/styles/styles';
 
 const Logout = () => {
-  const [user, setUser] = useContext(UserContext);
+  const [loading, setLoading] = useState(true);
+  const classes = formStyles();
   useEffect(() => {
-    setUser(false);
-  }, [setUser]);
+    const logout = async () => {
+      const accessToken = Cookies.get('accessToken');
```

Comment:
We shouldn't be storing cookies using a library - it should be sent automatically by the browser

Reply:
Misunderstood the docstring. Now works with browser cookies